gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -352,11 +336,13 @@ pub enum ClnExtensionError {
 }
 
 /// BOLT 4: https://github.com/lightning/bolts/blob/master/04-onion-routing.md#failure-messages
-/// 2002 error code reports general temporary failure of this processing node.
-fn temp_node_failure() -> serde_json::Value {
+/// 16399 error code reports unknown payment details.
+///
+/// TODO: We should probably use a more specific error code based on htlc processing fail reason
```

Comment:
Do you want that TODO in the rendered docs?

Reply:
`htlc_processing_failure` failure message is not part of  public gateway api. Doesn't need to be in rendered docs